lib/diagnostic.rb

```diff
@@ -108,10 +108,17 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash =
+class Person
+  def initialize(favorite_number, first_name)
+    @favorite_number = favorite_number
+    @first_name = first_name
+  end
+end
+first_person = new.Person(3, 'Dee Dee')
```

Comment:
Whoa looks like you overthought this one just a bit. We were looking for short hand syntax of creating a Hash:

`'person = {first_name: "Dee Dee", favorite_number: 3}'`